http://drm.bkbits.net/drm-2.6
airlied@starflyer.(none)|ChangeSet|20040829110011|43116 airlied

# This is a BitKeeper generated diff -Nru style patch.
#
# ChangeSet
#   2004/08/29 21:00:11+10:00 airlied@starflyer.(none) 
#   Stop i830 and i915 being built at the same time into the kernel
#   Provide better info on which one is needed where...
#   
#   Signed-off-by: Dave Airlie <airlied@linux.ie>
# 
# drivers/char/drm/Kconfig
#   2004/08/29 20:56:58+10:00 airlied@starflyer.(none) +6 -4
#   Stop i830 and i915 being built at the same time into the kernel
#   Provide better info on which one is needed where...
#   
#   Signed-off-by: Dave Airlie <airlied@linux.ie>
# 
diff -Nru a/drivers/char/drm/Kconfig b/drivers/char/drm/Kconfig
--- a/drivers/char/drm/Kconfig	2004-08-30 17:29:32 -07:00
+++ b/drivers/char/drm/Kconfig	2004-08-30 17:29:32 -07:00
@@ -57,22 +57,24 @@
 
 config DRM_I830
 	tristate "Intel 830M, 845G, 852GM, 855GM, 865G"
-	depends on DRM && AGP && AGP_INTEL
+	depends on DRM && AGP && AGP_INTEL && !(DRM_I915=y)
 	help
 	  Choose this option if you have a system that has Intel 830M, 845G,
 	  852GM, 855GM or 865G integrated graphics.  If M is selected, the
 	  module will be called i830.  AGP support is required for this driver
 	  to work. This driver will eventually be replaced by the i915 one.
+	  This driver should be used for systems running Xorg 6.7 and XFree86 4.4
+	  or previous releases.
 
 config DRM_I915
 	tristate "Intel 830M, 845G, 852GM, 855GM, 865G, 915G"
-	depends on DRM && AGP && AGP_INTEL
+	depends on DRM && AGP && AGP_INTEL && !(DRM_I830=y)
 	help
 	  Choose this option if you have a system that has Intel 830M, 845G,
 	  852GM, 855GM 865G or 915G integrated graphics.  If M is selected, the
 	  module will be called i915.  AGP support is required for this driver
-	  to work. This driver will eventually replace the I830 driver, when
-	  later release of X start to use the new DDX and DRI.
+	  to work. This driver should be used for systems running Xorg 6.8 and
+	  XFree86 releases after (but not including 4.4).
 	
 
 config DRM_MGA
